pysatNASA/instruments/methods/cdaweb.py

```diff
@@ -789,7 +859,7 @@ def list_remote_files(tag='', inst_id='', start=None, stop=None,
                 else:
                     search_times = pds.date_range(start, stop
                                                   + pds.DateOffset(years=1),
-                                                  freq='Y')
+                                                  freq='YE')
```

Comment:
Similar change with pandas 2.2

Reply:
Thanks for adding a version check for the pandas stuff.